Source/WebKit/NetworkProcess/cache/NetworkCacheKey.cpp

```diff
@@ -89,20 +89,20 @@ static void hashString(SHA1& sha1, const String& string)
     if (string.is8Bit() && string.containsOnlyASCII()) {
         const uint8_t nullByte = 0;
         sha1.addBytes(string.span8());
-        sha1.addBytes(&nullByte, 1);
+        sha1.addBytes(std::span { &nullByte, 1 });
         return;
     }
     auto cString = string.utf8();
     // Include terminating null byte.
-    sha1.addBytes(cString.dataAsUInt8Ptr(), cString.length() + 1);
+    sha1.addBytes(std::span { cString.dataAsUInt8Ptr(), cString.length() + 1 });
```

Comment:
Same comment as earlier, no need for the `cString` local and should rely on `Cstring::spanIncludingNullTerminator()`

Reply:
Will fix.